content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -0,0 +1,172 @@
+---
+reviewers:
+title: Role Based Access Control Good Practices
+description: >
+  Principles and practices for good RBAC design for cluster operators.
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+Kubernetes {{< glossary_tooltip text="RBAC" term_id="rbac" >}} is a key security control 
+to ensure that cluster users and workloads have only the access to resources required to 
+execute their roles. It is important to ensure that, when designing permissions for cluster
+users, the cluster administrator understands the areas where privilge escalation could occur, 
+to reduce the risk of excessive access leading to security incidents.
+
+The good practices laid out here should be read in conjunction with the general [RBAC documentation](/docs/reference/access-authn-authz/rbac/#restrictions-on-role-creation-or-update).
+
+<!-- body -->
+
+## General good practice
+
+### Least privilege
+
+Ideally minimal RBAC rights should be assigned to users and service accounts. Only permissions 
+explicitly required for their operation should be used. Whilst each cluster will be different, 
+some general rules that can be applied are :
+
+ - Assign permissions at the namespace level where possible. Use RoleBindings as opposed to 
+   ClusterRoleBindings to give users rights only within a specific namespace.
+ - Avoid providing wildcard permissions when possible, especially to all resources.
+   As Kubernetes is an extensible system, providing wildcard access gives rights
+   not just to all object types presently in the cluster, but also to all future object types
+   which are created in the future.
+ - Administrators should not use `cluster-admin` accounts except where specifically needed. 
+   Providing a low privileged account with impersonation rights can avoid accidental modification of 
+   cluster resources.
+ - Avoid adding users to the `system:masters` group. Any user who is a member of this group 
+   bypasses all RBAC rights checks and will always have cluster-admin access, which cannot be 
+   revoked by removing Role Bindings or Cluster Role Bindings. As an aside, if a cluster is 
+   using an authorization webhook, membership of this group also bypasses that webhook (requests 
+   from users who are members of that group are never sent to the webhook)
+
+### Minimize distribution of privileged tokens
+
+Ideally, pods shouldn't be assigned service accounts granted powerful permissions (listed [here](#Kubernetes-RBAC---Privilege-Escalation-Risks)). 
+In cases where a workload requires powerful permissions, consider the following practices:
+
+ - Limit the number of nodes running powerful pods. Ensure that any DaemonSets you run
+  are necessary and are run with least privilege.
+   nodes are least-privileged and to limit the blast radius of container escapes.
```

Comment:
copy/paste or sentence fragment?

Reply:
yeah  I think that one got a bit messed up somewhere. The sentence should be `Ensure that any DaemonSets you run
  are necessary and are run with least privilege to limit the blast radius of container escapes` . Updated.